docs/deployment/manage-projects.md

```diff
@@ -36,13 +36,14 @@ To sort projects, click the attribute you want to sort by on the header row. You
 Sort by the following attributes:
 
 - **Project**: Click to toggle between sorting project names alphabetically in ascending or descending order. 
-- **Last scan**: Click to toggle between sorting the projects' latest scans in ascending or descending order.
+- **Last scan**: Click to toggle between sorting the projects' latest scans in ascending or descending order. The sorting is based on when the last scan **started**, regardless of its status. For this reason, you may see that scans with statuses such as **Not started** or **Never finished** are not necessarily grouped together.
```

Comment:
```suggestion
- **Last scan**: Click to toggle between sorting the projects' latest scans in ascending or descending order. The sorting is based on when the last scan **started**, regardless of its status. For this reason, you may see scans with statuses such as **Not started** or **Never finished** not grouped together.
```

Reply:
I think I like Sara's version of this a bit better since "see scans [not grouped together]" is a bit weird if you take out the intervening words. But I don't feel strongly about it.

Reply:
That's fair; I also don't have strong feelings, I just commented since it took me a bit to parse the sentence. Could definitely just be me, though!